charts/rancher-vsphere-csi/values.yaml

```diff
@@ -145,37 +145,37 @@ global:
 # https://docs.vmware.com/en/VMware-vSphere-Container-Storage-Plug-in/3.0/vmware-vsphere-csp-getting-started/GUID-D4AAD99E-9128-40CE-B89C-AD451DA8379D.html#kubernetes-versions-compatible-with-vsphere-container-storage-plugin-1
 versionOverrides:
   # Versions from https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/release-3.0/manifests/vanilla/vsphere-csi-driver.yaml
-  - constraint: ">= 1.24 < 1.28"
+  - constraint: ">= 1.24 < 1.29"
```

Comment:
According to https://docs.vmware.com/en/VMware-vSphere-Container-Storage-Plug-in/3.0/rn/vmware-vsphere-container-storage-plugin-30-release-notes/index.html `v3.1.0` (and thus likely `v3.1.2`) officially supports v1.26 to v1.28 so we can't have a range this wide for this version and should break this range in two - v1.24 to v1.25 to remain using v3.0.2 and new v1.26 to v1.28 range to use newly introduced v3.1.2.

Reply:
I wondered about that when doing this - I split up the range for 3.1.2 to support 1.26-1.28 and ended 3.0.2 support at 1.25.x